util/paywall.go

```diff
@@ -110,6 +119,16 @@ func makeRequest(url string, timeout time.Duration) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	if response.StatusCode != http.StatusOK {
+		body, err := ioutil.ReadAll(response.Body)
+		if err != nil {
+			return nil, fmt.Errorf("   *** ERROR invalid "+
```

Comment:
Don't use `*** Error`. That is specifically for dcrtime for parsing reasons. Say something like
`"dcrdata error: %v %v" %v", response.StatusCode, route, body`